packages/core/test/node/com.spec.ts

```diff
@@ -160,6 +160,25 @@ describe('Communication', () => {
         expect(handleMessageStub).to.have.not.been.called;
     });
 
+    it('serialization of arguments for remote API calls should work as expected ', async () => {
```

Comment:
"should work as expected" does not really help to understand what is expected

Reply:
@olehrakwix fixed